test/functionalTest/cases/2750_common_metrics/2750_incoming_errors.test

```diff
@@ -0,0 +1,519 @@
+#   - No accept header
+#   - Content-Length > PAYLOAD_MAX_SIZE
+#   - no acceptable mime-type in accept header
+#   - contentType != "" for GET request
+#   - contentType != "" for DLETE request
+#   - PUT/POST/PATCH without payload
+#   - Empty right-hand-side for URI param
+#   - Bad pagination offset - must be a decimal number
+#   - Bad pagination limit - must be a decimal number
+#   - Bad pagination limit > 1000 (MAX_PAGINATION_LIMIT)
+#   - Bad pagination limit == 0
+#   - Bad value for /details/   (details=badValueForDetails)
+#   - q missing in accept header
+#   - missing equal-sign after q in accept header
+#   - qvalue in accept header is missing
+#   - qvalue in accept header is not a number
+#   - Invalid mediaRange (see acceptHeadersAcceptable)
+#   - Invalid out-format (see ciP->httpHeaders.outformatSelect())
+#   - Invalid value for URI param /options/
+#   - invalid character in URI parameter
+#   - Only /absolute/ Service Paths allowed [a service path must begin with /]
+#   - too many components in ServicePath
+#   - component-name too long in ServicePath
+#   - a component of ServicePath contains an illegal character
+#   - too many service paths - a maximum of ten service paths is allowed
+#   - Content-Type header not used, default application/octet-stream is not supported
+#   - api V1 && contentType != "application/json"
+#   - api V2 && contentType != "application/json" && contentType != "text/plain"
+#   - forbiddenChars in URI
+#
+#   + all service routines that sets a ciP->httpStatusCode >= 300 ... Puhhhh!
+#     Probably thousands of possibler causes for errors. Impossible to test each and every one. Lets pick ONE serviceRoutine only
+#     [ PUT /v2/entities/<id>/attrs/<attrName>/value]:
+#
+#   - forbiddenChars in entityId
+#   - forbiddenChars in attributeName
+#   - attribute name length is ZERO
+#   - attribute name length > MAX_ID_LEN
+#   - error in compound value
+#   - forbiddenChars in string value (if a string ...)
+#   - more than one service path
+#   - other errors in service path
+#  
+
+#
+# 00. GET /v2/entities
+# 01. Ask for metrics and see no errors
+# 02. Send a request with an invalid JSON payload
+# 03. Ask for metrics and see one error
+# 04. Send a request with an Invalid URL
+# 05. Send a request with an Invalid Service Path
+# 06. Send a request with an Invalid Content-Type
+# 07. Send a request with invalid value for URI param /options/
+# 08. Send a request with Content-Length > PAYLOAD_MAX_SIZE
+# 09. Send a request with no acceptable mime-type in accept header
+# 10. Send a request with an invalid character in a URI parameter
+# 11. Send a request with too many components in ServicePath
+# 12. Send a PUT /v2/entities/<id>/attrs/<attrName>/value for an non-existing entity E_NOPE
+# 13. Ask for metrics and see 10 errors
+#
+
+
+echo "00. GET /v2/entities"
+echo "===================="
+orionCurl --url /v2/entities --servicePath /entities
```

Comment:
This is the same comment as the one above.
If you really don't like this (just, take a look at the functests without diff-noise first, please), I can change it.
However, I prefer to keep it as it is (which is why I implemented it this way).